docs/src/design-system/building-blocks/button/ButtonStatesExample.tsx

```diff
@@ -114,7 +114,7 @@ const ButtonStatesExample: React.FC<Props> = ({ variantType, overrideTheme }) =>
               <Typography variant="subheader3">Active/Pressed</Typography>
             </Grid>
             <Grid item md={1} sx={{ paddingLeft: paddingLeft }}>
-              <Typography variant="subheader3">Focused(ADA)</Typography>
+              <Typography variant="subheader3">focused(ADA)</Typography>
```

Comment:
`text-transform: capitalize` applies to this element, so making it lower case will not show up.

Reply:
Yes, you're correct! I think I was making some other tests during this PR , and forgot to remove it.